docs/reference/aggregations/metrics/percentile-rank-aggregation.asciidoc

```diff
@@ -51,8 +51,8 @@ The response will look like this:
    "aggregations": {
       "load_time_ranks": {
          "values" : {
-            "500.0": 55.00000000000001,
-            "600.0": 64.0
+            "500.0": 90.01,
+            "600.0": 100.0
```

Comment:
While these changes bring this snippet in line with its surrounding paragraphs, it fails our CI that runs the actual test. 

The data for this test comes from  the `// TEST[setup:latency]` comment in the preceding snippet. You can see how the data is generated here:
https://github.com/elastic/elasticsearch/blob/master/docs/build.gradle#L484

To make this snippet work with your changes, add the following comments to line 62 after the `// TESTRESPONSE[...` comment:
```
// TESTRESPONSE[s/"500.0": 90.01/"500.0": 55.00000000000001/]
// TESTRESPONSE[s/"600.0": 100.0/"600.0": 64.0/]
```

If you don't mind, you make similar changes to the response in the 'Keyed Response' section.

Reply:
Here's a commit containing my recommended changes if that's easier to follow:
59845800918804bb0146e67d641b8a0f11d69dfb